### PR TITLE
Item 6709: CDS bug fixes related to ETL, report lag, treatment filter, url encoding, & buttons display at different screen resolutions

### PR DESCRIPTION
**Pull request by labkey-bpatel**
Ticket 39344: ETL error message - cvd264 in studyrelationship table
Ticket 39119: Lag opening report when clicking on mAb report buttons
Ticket 39104: Active filters not applying treatment
Ticket 39351: mAb 'MVN/A12' link defaults to 'MVN' mAb page
Ticket 39120: MAb right most report button hidden at different zoom levels

### CODE REVIEW

File: webapp/Connector/src/view/MabGrid.js [hunk 1]
```diff
@@ -720,6 +726,18 @@ Ext.define('Connector.view.MabGrid', {
             return false;
         }
 
+        // Fix for Issue 39229: Lag opening report when clicking on mAb report buttons
+        // Immediately go to the report view before prepareMAbReportQueries step below instead of staying on the grid view
+        // This report view is a temporary one, and will be later removed
```
Comment (XingY) [inline webapp/Connector/src/view/MabGrid.js]: Can Connector.getQueryService().prepareMAbReportQueries be moved to mabreportview itself so we don't need to do the temp view thing?

**Review by XingY — APPROVED** [review]
Would be good if mabreportview itself handles loading, fine to leave it as it is.

File: webapp/Connector/src/view/MabGrid.js (hunk 1 above)
Comment (labkey-bpatel) [reply webapp/Connector/src/view/MabGrid.js]: Agreed! now done!

**Review by XingY — APPROVED** [review]

File: webapp/Connector/src/model/Measure.js [hunk 2]
```diff
@@ -47,6 +47,9 @@ Ext.define('Connector.model.Measure', {
         {name: 'isHoursType', type: 'boolean', defaultValue: false},
         {name: 'allowHoursTimePoint', type: 'boolean', defaultValue: false},
 
+        // Property to skip displaying in plot tooltip
```
Comment (XingY) [inline webapp/Connector/src/model/Measure.js]: "Property" -> "Array of properties"

File: webapp/Connector/src/utility/PlotTooltip.js [hunk 3]
```diff
@@ -197,8 +197,12 @@ Ext.define('Connector.utility.PlotTooltip', {
     buildPointAxisDetailTooltip: function(plot, axis, record, aggregators, hierarchicalDimensionInfo) {
         var content = '<div class="axis-details">', colon = ': ', linebreak = '<br/>';
         if (axis) {
+            var skipToolTip = axis.skipToolTip;
             var dimensions = axis.options.dimensions;
             for (var dim in dimensions) {
+                if (skipToolTip && skipToolTip.length > 0 && skipToolTip.indexOf(dim) >= 0) {
```
Comment (XingY) [inline webapp/Connector/src/utility/PlotTooltip.js]: Could use Ext.isArray to check instead, string also has length and indexOf, which might be confusing

**Review by XingY — APPROVED** [review]
Just some minor comments

File: webapp/Connector/src/model/Measure.js (hunk 2 above)
Comment (labkey-bpatel) [reply webapp/Connector/src/model/Measure.js]: Done!

File: webapp/Connector/src/utility/PlotTooltip.js (hunk 3 above)
Comment (labkey-bpatel) [reply webapp/Connector/src/utility/PlotTooltip.js]: Excellent point! Done!

**Pull request merged**